ghcide/ghcide.cabal

```diff
@@ -1,20 +1,18 @@
-cabal-version:      3.4
-build-type:         Simple
-category:           Development
-name:               ghcide
-version:            2.8.0.0
-license:            Apache-2.0
-license-file:       LICENSE
-author:             Digital Asset and Ghcide contributors
-maintainer:         Ghcide contributors
-copyright:          Digital Asset and Ghcide contributors 2018-2020
-synopsis:           The core of an IDE
-description:        A library for building Haskell IDE's on top of the GHC API.
-homepage:
-  https://github.com/haskell/haskell-language-server/tree/master/ghcide#readme
-
-bug-reports:        https://github.com/haskell/haskell-language-server/issues
-tested-with:        GHC ==9.8.2 || ==9.6.5 || ==9.4.8 || ==9.2.8
+cabal-version: 3.4
+build-type: Simple
+category: Development
+name: ghcide
+version: 2.8.0.0
+license: Apache-2.0
+license-file: LICENSE
+author: Digital Asset and Ghcide contributors
+maintainer: Ghcide contributors
+copyright: Digital Asset and Ghcide contributors 2018-2020
+synopsis: The core of an IDE
+description: A library for building Haskell IDE's on top of the GHC API.
+homepage: https://github.com/haskell/haskell-language-server/tree/master/ghcide#readme
```

Comment:
Which version of Gild was used here? As of [1.2.0.0](https://github.com/tfausak/cabal-gild/releases/tag/1.2.0.0) it shouldn't have re-formatted this line. (The URL should've stayed on its own line after `homepage:`.)

Reply:
1.1.1.0 was used, will reformat with 1.2.0.0, thanks for the hint!